packages/cli/test/integration/credentials.controller.test.ts

```diff
@@ -24,6 +24,7 @@ beforeEach(async () => {
 
 type GetAllResponse = { body: { data: ListQuery.Credentials.WithOwnedByAndSharedWith[] } };
 
+// NOTE: passing
```

Comment:
We might as well remove those `NOTE`s since they are probably no longer necessary